### DISCUSSION TITLE
Option to select daily granularity

### DISCUSSION THREAD

**Discussion by mattiasback** [davetapley/helium-tax#71] [Q&A] [answered]
I would like the option to sum the rewards by daily granularity in order to reduce numbers of entries from 1000's to 365.

Thanks for a great tool.

**Comment by davetapley** [comment, marked as answer]
You can actually do this all in [Google Sheets](http://sheets.google.com/) using a pivot table!

1. Import `.csv`
2. Add new column to right of column A
3. In blank cell B2 enter formula `=DATEVALUE(MID(A2,1,10)` and accept autofill
4. Select 'Insert' then 'Pivot Table'
5. On new pivot table:
  a. Under 'Rows' select 'Column B'
  b. Under 'Values' select 'USD'


It should look like this:

<img width="806" alt="helium-tax_pivot_table_-_Google_Sheets" src="https://user-images.githubusercontent.com/48232/155003456-cb5d876c-ac9e-4ec9-9bf9-ccf29dbd7e28.png">
